AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -32,6 +32,7 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 astv.ru###overlvaysubcribe
 astv.ru###followwhatsapp
 miloserdie.ru##.adp-popup
+bloombergquint.com##.pn-boilerplate--newsletter
```

Comment:
It doesn't work on my end. Why not `bloombergquint.com###bq-piano-story-page-newsletter`?